### ISSUE TITLE
Few bugs on Raspberry Pi with 0.6.27-4k

### ISSUE THREAD

**Issue by kristofejro** [pnp4nagios/pnp4nagios#50]
Just wanted to add my 5 cents to the project.
I wanted to try new version and tried to install latest version:
https://github.com/pnp4nagios/pnp4nagios/archive/refs/tags/v0.6.27-4k.tar.gz

First issue I have found was during `make install`

```
make[1]: Entering directory '/tmp/pnp4nagios-0.6.27-4k/src'
make install-basic
make[2]: Entering directory '/tmp/pnp4nagios-0.6.27-4k/src'
/usr/bin/install -c -m 755 -d /usr/local/bin
/usr/bin/install -c -m 755 -o nagios -g nagios -d /usr/local/usr/lib/pnp4nagios
/usr/bin/install -c -m 755 -o nagios -g nagios -d /usr/local/var/log/pnp4nagios
/usr/bin/install -c -m 755 -o nagios -g nagios -d /usr/local/pnp4nagios/var/perfdata
/usr/bin/install -c -m 775 -o nagios -g nagios -d /usr/local/pnp4nagios/var/spool
/usr/bin/install -c -m 755 npcd /usr/local/bin
/usr/bin/install -c -m 754 -o nagios -g nagios npcdmod.o /usr/local/usr/lib/pnp4nagios
make[2]: Leaving directory '/tmp/pnp4nagios-0.6.27-4k/src'
make strip-post-install
make[2]: Entering directory '/tmp/pnp4nagios-0.6.27-4k/src'
/usr/bin/strip /usr/local/bin/npcd
/usr/bin/strip /usr/local/usr/lib/pnp4nagios/npcdmod.o
make[2]: Leaving directory '/tmp/pnp4nagios-0.6.27-4k/src'
make[1]: Leaving directory '/tmp/pnp4nagios-0.6.27-4k/src'
cd ./man && make install
make[1]: Entering directory '/tmp/pnp4nagios-0.6.27-4k/man'
/usr/bin/install -c -m 755 -o nagios -g nagios -d /usr/local/man/man8
/usr/bin/install -c -m 755 -o nagios -g nagios npcd.8 /usr/local/man/man8
make[1]: Leaving directory '/tmp/pnp4nagios-0.6.27-4k/man'
cd ./share && make install
make[1]: Entering directory '/tmp/pnp4nagios-0.6.27-4k/share'
/usr/bin/install -c -m 755 -o apache -g apache -d /usr/local/var/log/pnp4nagios/kohana
/usr/bin/install: invalid user ‘apache’
make[1]: *** [Makefile:27: install] Error 1
make[1]: Leaving directory '/tmp/pnp4nagios-0.6.27-4k/share'
make: *** [Makefile:187: install] Error 2
```

My apache is not running in context od `apache` user but in context od `www-data` user so I have done some changes in `config.status`

```
#S["COMMAND_OPTS"]="-o apache -g apache"
S["COMMAND_OPTS"]="-o www-data -g www-data"
#S["http_grp"]="apache"
#S["http_user"]="apache"
S["http_grp"]="www-data"
S["http_user"]="www-data"
```
And it helped to go through the installation.

Second issue was that Nagions was complaining:

```
Dec 29 14:42:31 raspberrypi nagios[25026]: Warning: File '/usr/local/pnp4nagios/var/host-perfdata' could not be opened - host performance data will not be written to file!
Dec 29 14:42:31 raspberrypi nagios[25026]: Warning: File '/usr/local/pnp4nagios/var/service-perfdata' could not be opened - service performance data will not be written to file!
```

I have found that the installer is assigning wrong permissions:

```
/usr/local/pnp4nagios# ls -l
total 16
drwxr-sr-x 7 nagios nagios 4096 Dec 29 15:33 etc
drwxr-sr-x 8 nagios nagios 4096 Dec 29 15:33 share
drwxr-sr-x 4 root   staff  4096 Dec 29 15:33 var
```
When changed `chown -R nagios:nagios /usr/local/pnp4nagios/var` nagios and npcd started to work.
All performance data was appended in the folder.
pnp4nagios web page was displayed and it passed all tests.

Last issue is that it does not display any graphs:
![obraz](https://github.com/pnp4nagios/pnp4nagios/assets/15168170/8ef648ab-23b7-41c2-b34a-b9503b9cf941)

When opening graph in separate tab there is an error:
![obraz](https://github.com/pnp4nagios/pnp4nagios/assets/15168170/e593d6fb-7538-42be-97ee-8d3eb2fb3d32)

System:
 - NagiosCore : 4.5.0
 - OS: Raspbian (Bookworm)
 - Browser: Firefox
 - Version: 121.0
 - PHP 8.2

**Comment by sistemmsn** [comment]
Downgrade your PHP version to 8.0 if you want it to work, In what they solve the errors

**Comment by celane** [comment]
At least some of these issues should be solved in the new 0.6.27-5 release in the 'main' branch (testing branch merged in), maybe all of the issues, since they sure seem like familiar problems that got solved in testing iterations. 

But I don't have a Raspberry Pi to test on, so please test if you can.
Closing the issue, open a new one if there problems with the new release.
